src/python/WMCore/REST/Main.py

```diff
@@ -619,5 +625,8 @@ def main():
             else:
```

Comment:
Perhaps the option above in the `if` statement should be removed and we define this option to be in the format of `-l my_logfile.log` only. In other words, `|`, `rotatelogs` and `86400` (integer with time in seconds) are no longer an acceptable option. What do you think?